### PR TITLE
 [bugfix/PLAYER-2698] Video 360 is navigated instead of page scrolling

### PR DESCRIPTION
**Pull request by IvanSakharovskii**

### CODE REVIEW

File: js/views/playingScreen.js [hunk 1]
```diff
@@ -141,14 +143,27 @@ var PlayingScreen = React.createClass({
...
-    else if (!this.props.controller.videoVr) {
-      this.props.controller.togglePlayPause(event);
+    if (event.target.className === "oo-state-screen-selectable") {
+      event.preventDefault();//to prevent mobile from propagating click to discovery shown on pause
+      if (!this.state.controlBarVisible){
+        this.showControlBar(event);
+        this.props.controller.startHideControlBarTimer();
+      }
+      else {
+        var isNeedToggle = false;
```
Comment (pilievOoyala) [inline js/views/playingScreen.js]: Can you change this to var shouldToggle?

**Comment by pilievOoyala** [comment]
You can do the variable name change in another pull require. Otherwise LGTM

**Comment by jpfranco** [comment]
LGTM but can we add a unit test that covers the changes?

File: js/views/playingScreen.js (hunk 1 above)
Comment (KseniiaSokolovaEpam) [reply js/views/playingScreen.js]: \+

**Comment by KseniiaSokolovaEpam** [comment]
One test was changed and there is no new functional for new tests.

File: js/skin.js [hunk 2]
```diff
@@ -123,19 +122,30 @@ var Skin = React.createClass({
    * @public
    * @description the function is called when we stop the rotation
    */
-  handleVrPlayerMouseUp: function() {
+  handleVrPlayerMouseUp: function(e) {
```
Comment (aeng7) [inline js/skin.js]: Can you add the new argument in the jsdoc?

File: js/skin.js [hunk 3]
```diff
@@ -123,19 +122,30 @@ var Skin = React.createClass({
...
     if (this.props.controller && this.props.controller.isVrStereo) {
       return;
     }
     if (this.props.controller && this.props.controller.videoVr) {
+      var isVrMouseMove = this.state.isVrMouseMove;
+      if (Utils.isIos()) {
+        isVrMouseMove = false;
```
Comment (aeng7) [inline js/skin.js]: How come we are setting this to false on iOS?

File: js/skin.js [hunk 4]
```diff
@@ -123,19 +122,30 @@ var Skin = React.createClass({
...
+        isVrMouseMove: isVrMouseMove,
         xVrMouseStart: 0,
         yVrMouseStart: 0
       });
+
       if (typeof this.props.controller.checkVrDirection === 'function') {
         this.props.controller.checkVrDirection();
       }
+
+      var endMove = this.state.isVrMouseMove || OO.isAndroid || OO.isIos;
```
Comment (aeng7) [inline js/skin.js]: Can we add a comment about why this is true for Android and iOS?

File: js/skin.js (hunk 2 above)
Comment (KseniiaSokolovaEpam) [reply js/skin.js]: \+

File: js/skin.js (hunk 3 above)
Comment (KseniiaSokolovaEpam) [reply js/skin.js]: \+

File: js/skin.js (hunk 4 above)
Comment (yvinokurov) [reply js/skin.js]: +

Comment (yvinokurov) [reply js/skin.js]: add comment

**Comment by pilievOoyala** [comment]
LGTM

**Comment by jpfranco** [comment]
LGTM

**Pull request merged**